### PR TITLE
 Fix problems in focus mode and global pattern switching 

### PR DESCRIPTION
**Pull request by MMMMoriaty**

### CODE REVIEW

File: packages/lb-annotation/src/core/pointCloud/store/index.ts [hunk 1]
```diff
@@ -333,9 +339,8 @@ class PointCloudStore {
     }
 
     const polygon = originPolygon;
-    const originPoints = this.scene.getObjectByName(this.pointCloudObjectName) as THREE.Points;
 
-    const cloudDataArrayLike = originPoints?.geometry?.attributes?.position?.array;
+    const cloudDataArrayLike = this.originPoints;
```
Comment (Kerwin-L) [inline packages/lb-annotation/src/core/pointCloud/store/index.ts:343]: What issue did you fix?

Comment (MMMMoriaty) [reply packages/lb-annotation/src/core/pointCloud/store/index.ts:343]: origin pointcloud data was removed in focus mode, so you can't add or remove points in focus mode. 
Now i add origin points into store cache when loading pcd file to fix this problem

**Pull request merged**